### DISCUSSION TITLE
项目是否涉及政治

### DISCUSSION THREAD

**Discussion by user-hook-name** [WeNeedHome/SummaryOfLoanSuspension#567] [Ideas]
## 表象:
1. 项目全网屏蔽
2. 删除发起人知乎账号
3. 国内媒体引用项目数据缺未标注来源

## 态度已经明确:
1. 项目违法
2. 参与项目人员追求法律责任
3. 所谓不要谈有关政治=逃避

**Comment by zxcs001** [comment]
随意，最好把我抓进去，关个一两年。

然后我扭头申请美国美国政治避难，在美国搞码农多香……

你用这种东西威胁一个已经不在乎征信，不在乎那些乱七八糟东西的人，有意义么？

共产党那套东西，只能威胁一下，收入不错，有体面生活的体面人。真的遇见烂尾，背上几十年贷款拿不到房，还要租房过的连狗的不如的人。

谁搭理你那些东西啊，懂了么，五毛？

**Reply by user-hook-name** [reply]
所以项目本身就非法,属于高度政治问题.
为啥很多人还在自我阉割,不讨论政治问题.

**Reply by yalo-ddaa** [reply]
楼主是友军啊……

**Reply by user-hook-name** [reply]
> 楼主是友军啊……

亦敌亦友

**Comment by yalo-ddaa** [comment]
在国内统计也违法

**Comment by C-Joey** [comment]
1.就算不涉及任何政治，这个项目也会被屏蔽（断供不利于维稳，增大金融风险，会涉及到问责，会给境外“递刀子”）
2.删除发起人知乎账号也是必然的（一切不利于稳定的因素都会被销号，国内的社交媒体你难道不知道什么下限吗，都是网信办一句话）
3.国内媒体都是维稳工具，不能唱衰的，你以为是真想帮忙吗。

**Reply by user-hook-name** [reply]
对政治敏感度的无穷大和非友即敌的表现,感到可悲

**Reply by C-Joey** [reply]
> 对政治敏感度的无穷大和非友即敌的表现,感到可悲

所以你不是友军吗

**Reply by ruixiaojia** [reply]
只想维稳不想解决问题，总有一天会稳不住
肯定要涉及到问责，怎么导致这么大的问题发生的难道不该问责吗？
自家人都“掏刀子”，还怕给境外“递刀子”？

**Reply by user-hook-name** [reply]
> > 对政治敏感度的无穷大和非友即敌的表现,感到可悲
> 
> 所以你不是友军吗

反对极端主义,那种一味批评政府一无是处.更讨厌那种,所谓那种不要谈政色变,如过街老鼠人人喊打而诛之的新时代的愚忠.

**Reply by user-hook-name** [reply]
> 只想维稳不想解决问题，总有一天会稳不住 肯定要涉及到问责，怎么导致这么大的问题发生的难道不该问责吗？ 自家人都“掏刀子”，还怕给境外“递刀子”？

其实国内已经枪口统一.项目就是境外分子干的.然后这些所谓被"境外分子"的人,还在找他们的"境外分子".
哎.何苦了.

**Comment by zongjiangU** [comment]
谈论政治算违法是什么道理。
妄议朝政罪吗。
几千年刻在DNA里的对官府的恐惧。

**Reply by user-hook-name** [reply]
奴性

**Comment by program-thinked** [comment]
政治问题不涉及政治这是脑子不好使。

**Reply by user-hook-name** [reply]
从根本否定自己
让我想到水浒传

**Comment by b4o** [comment]
本来正常的，可是后来进来了老鼠，影响了更多的人

**Comment by woshizengling** [comment]
老鼠太多了

**Comment by dalezhang93** [comment]
躲避政治本身就是一种政治态度,避不开的

**Reply by user-hook-name** [reply]
那些躲避政治的人.就好像有人和他讨论三上悠亚时候,那种鄙视的感觉.其实自己门清现在的状况.

**Comment by JusticeEER** [comment]
> ## 表象:
> 1. 项目全网屏蔽
> 2. 删除发起人知乎账号
> 3. 国内媒体引用项目数据缺未标注来源
> 
> ## 态度已经明确:
> 1. 项目违法
> 2. 参与项目人员追求法律责任
> 3. 所谓不要谈有关政治=逃避

国内到处是政治，你叫他不讲政治，他就说你是汉奸

**Comment by lance111l** [comment]
项目违法 就该整治   政治太模糊  政治通过政策法律来体现
